dev/droidmate/.idea/modules/projects/uiautomator2-daemon/uiautomator2-daemon.iml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This entire .iml file should not exist in repository. It is ignored by droidmate\dev\.gitignore. Delete from repo. It will be auto-regenerated on "Refresh all gradle projects"